### DISCUSSION TITLE
Translation improvements - Share the steps to do

### DISCUSSION THREAD

**Discussion by bovirus** [rustdesk/rustdesk#4065] [General]
### Description

@rustdesk 

Before to make the changss i 'd like to share with you the improvements steps (different PR) for translation

**1. step**

Add strings 

"Default"
"System Sound"

to template.rs and italian.rs files
The source code don't require changes because is already done to use these strings translated.
We need just to add the new strings to .rs files.

**2. step** 

Change the source code to use these strings (different then current)

"Default" -> "System language"
"Automatic" -> "Automatic codec"

to make the strings more specific.

**3. step**

Modify the source code and .rs file to translate the current untranslated string regarding login.

Thanks.

### Impact

Improve translation

### Additional Context

@rustdesk 

Before to make the chanegs i 'd like to share with you the improvements steps for translation

**1. step**

Add strings 

"Default"
"System Sound"

to template and italian.rs files
The source code don't require changes because is already done to use these strings translated.
We need just to add to .rs files.

**2. step** 

Change the source code to use these strings (different then current)

"Default" -> "System language"
"Automatic" -> "Automatic codec"

to make the strings more specific.

**3. step**

Modify the source code and .rs file to translate the current untranslated string regarding login.

Thanks.

**Comment by yurtpage** [comment]
You may try to send your own PR with these changes
